python/utils/request_util.py

```diff
@@ -26,9 +44,24 @@ def _create_backoff_decorator(max_tries: int, factor: int, max_time: int) -> Any
             max_time=max_time
         )
 
-    def run_request(self, uri: str, method: str, data: Any = None, params: Optional[dict] = None,
-                    factor: int = 15, content_type: Optional[str] = None,) -> requests.Response:
-        """Run request."""
+    def run_request(
+            self, uri: str, method: str, data: Any = None, params: Optional[dict] = None, factor: int = 15,
+            content_type: Optional[str] = None, accept_return_codes: list[int] = []) -> requests.Response:
```

Comment:
How do you feel about putting each of these args on a new line? I think it can make it easier to read